Jenkinsfile

```diff
@@ -56,33 +66,92 @@ def get_daos_packages(String distro) {
     return pkgs + "-" + daos_packages_version(distro)
 }
 
-def component_repos() {
+def pr_repos() {
     return cachedCommitPragma(pragma: 'PR-repos')
 }
 
-String el7_component_repos() {
+String el7_pr_repos() {
     return cachedCommitPragma(pragma: 'PR-repos-el7')
 }
 
-String leap15_component_repos() {
+String leap15_pr_repos() {
     return cachedCommitPragma(pragma: 'PR-repos-leap15')
 }
 
-def daos_repo() {
+String unit_repos() {
+    Map stage_info = parseStageInfo()
+    return unit_repos(stage_info['target'])
+}
+
+String unit_repos(String distro) {
+    if (distro == 'centos7') {
+        return el7_pr_repos() + ' ' + pr_repos()
+    }
+    if (distro == 'leap15') {
+        return leap15_daos_repos() + ' ' + pr_repos()
```

Comment:
Why is this `leap15_daos_repos()` when it's `el7_pr_repos()` on line 88.  Specifically `daos` vs. `pr`.